ilastik/applets/dataSelection/dataSelectionGui.py

```diff
@@ -601,9 +600,9 @@ def addStack(self, roleIndex, laneIndex):
         try:
             # FIXME: do this inside a Request
             self.parentApplet.busy = True
-            inner_path = stack_info.importAsLocalDataset(project_file=self.project_file)
+            inner_path = stack_info.importAsLocalDataset(project_file=self.get_project_file())
```

Comment:
Does new qt handle properties in a special way?

Reply:
yes

> The cause is PyQt5 accessing all class attributes via PyObject_GetAttr during initialization of the Gui Widget.
In our case, the class is not fully initialized by that time but class properties are still executed (via PyObject_GetAttr).